packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -333,10 +343,17 @@ const AccountSwitcher = props => {
     const getTotalDemoAssets = () => {
         const vrtc_loginid = props.account_list.find(account => account.is_virtual).loginid;
         const vrtc_balance = props.accounts[vrtc_loginid] ? props.accounts[vrtc_loginid].balance : 0;
+        const vrtc_currency = props.accounts[vrtc_loginid] ? props.accounts[vrtc_loginid].currency : 'USD';
         const mt5_demo_total = getTotalBalance(props.mt5_login_list);
         const dxtrade_demo_total = getTotalBalance(props.dxtrade_accounts_list);
 
-        let total = vrtc_balance;
+        let total;
+
+        if (vrtc_currency !== props.obj_total_balance.currency) {
+            total = vrtc_balance * exchanged_rate;
+        } else {
+            total = vrtc_balance;
+        }
```

Comment:
this condition can be simplified. 
```suggestion
        let total = vrtc_currency !== props.obj_total_balance.currency ? vrtc_balance * exchanged_rate : vrtc_balance;
```